internal/trace/listener.go

```diff
@@ -155,10 +160,17 @@ func startFunctionExecutionSpan(ctx context.Context, mergeXrayTraces bool, isDdS
 		resourceName = string(extension.DdSeverlessSpan)
 	}
 
+	var childOfOpt tracer.StartSpanOption
+	if parentSpanContext != nil {
+		if psc, ok := parentSpanContext.(*tracer.SpanContext); ok {
+			childOfOpt = tracer.ChildOf(psc) //nolint:SA1019
```

Comment:
This is actually deprecated but we needed because the span is not available here for us to use `Span.StartChild` because of in "Serverless Universal Instrumentation", that span is created by the agent instead of the tracer.